### ISSUE TITLE
EnTK hangs when pilot enters Allocating stage

### ISSUE THREAD

**Issue by mingtaiha** [radical-cybertools/radical.entk#171]
When trying to run my script on branch new_arch, I get the problem that the CUs enter `Allocating` state, but does not continue to `Executing` state. Running with `RADICAL_PILOT_VERBOSE=DEBUG` doesn't show anything out of the ordinary. I checked the json files and client logs do not show anything out of the ordinary either. This is most likely an issue on the RP layer, but I'm posting it here for anyone else using this EnTK branch.

**Comment by vivek-bala** [comment]
Hey Ming, can you run it with RADICAL_PILOT_VERBOSE=debug and RADICAL_PILOT_AGENT_VERBOSE=DEBUG and send the log please.

**Comment by mingtaiha** [comment]
Here's the client output [allocating_hold.txt](https://github.com/radical-cybertools/radical.ensemblemd/files/405050/allocating_hold.txt), and a zip of the session [rp.session.radical.mingtha.017019.0005-pilot.0000.zip](https://github.com/radical-cybertools/radical.ensemblemd/files/405051/rp.session.radical.mingtha.017019.0005-pilot.0000.zip)

**Comment by vivek-bala** [comment]
Can you check the zip file again please ? its recursively zipped. Also, if
you are attaching a single file/log, it might be more convenient to use a
gist (gist.github.com).

**Comment by mingtaiha** [comment]
Not recursive anymore

[rp.session.radical.mingtha.017019.0005-pilot.0000.zip](https://github.com/radical-cybertools/radical.ensemblemd/files/407375/rp.session.radical.mingtha.017019.0005-pilot.0000.zip)

**Comment by vivek-bala** [comment]
I don't see anything wrong with logs and can't reproduce this error (not stampede). Bringing in reinforcements, @andre-merzky , do you have any ideas ?

**Comment by andre-merzky** [comment]
Hi Vivek, Ming: this looks strange indeed.  Would you mind sending the session's json file?  What are the machines involved (client, agent)?   How was the run terminated?  What version of RP do you use?  Thanks!  Andre.

**Comment by mingtaiha** [comment]
Here's the JSON file [rp.session.radical.mingtha.017022.0001.json.zip](https://github.com/radical-cybertools/radical.ensemblemd/files/409524/rp.session.radical.mingtha.017022.0001.json.zip). This is from a run which I ran today. It exhibits the same behavior.

The client is the RADICAL VM (144.76.72.175), and the target machine is xsede.stampede. I terminated the run manually. The pilot just hangs once the CUs reach the allocating stage.
I am currently using RP version 0.41, the version that EnTK installs its dependencies.
